README.rst

```diff
@@ -72,6 +72,23 @@ If you don't want the errors, then make use of an extra argument to the command
 
 This last step should be performed every time you create new or make changes to existing programs.
 
+Show Programs Tab
+~~~~~~~~~~~~~~~~
+
+In order to show programs tab in the LMS dashboard, users will need to manually create an entry in ``Programs api config`` model in LMS Admin Panel. Go to ``Home > Programs >  Programs api config``. Add ``Marketing path`` equal to ``/programs`` and enable it. Then Programs tab will be shown on the LMS where users can view their registered programs. It will show like as in below picture.
```

Comment:
It is unclear from reading this is the user needs to add both Program API config and Site configuration. If both are needed, we can emphasize adding values for both and then show what will happen if we only add Programs API config. Please correct me if I am missing something here.

Reply:
Yes, these Program API config is needed if users wants to enable Programs page(This page will only show enrolled courses) on LMS, this image [link](https://github.com/overhangio/tutor-discovery/pull/58#:~:text=https%3A//private%2Duser,eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI0NjI2MzEsIm5iZiI6MTcwMjQ2MjMzMSwicGF0aCI6Ii80NjkyMzc3LzI3MDM2NTUyNC00YTcxNWZiYi05YzY2LTRiMWMtYTNjOS1mNDI5ODNmYjIyMmIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMTNUMTAxMjExWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NTY0YTgxZTIyZjA0MTRhMWIxZDRlZGM2OTY4ZmQxMmNlZTBlYjc5NjczZjFiMGU2MGIyYmIwMTNmYWM0ZmQ1MSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.vzPYOlqeLPg2gN1k%2D1H_u2L0hcdxjzUWRtCDUxleN3U) is broken, it was working before, I'll upload a new one for better understanding.

Site Configuration is needed if user have marketing site which shows all existing programs.

Reply:
These steps would be enough and useful if some users wants to show programs to end-users. It's there choice as we can only give steps.